tt_metal/hw/inc/dataflow_api.h

```diff
@@ -1522,12 +1531,27 @@ enum class BufferType: uint8_t {
     SYSTEM_MEMORY = 2
 };
 
+FORCE_INLINE
+uint32_t min(uint32_t a, uint32_t b) { return (a < b) ? a: b; }
+
+// one entry for num_pages in shard
+// one entry for core's x coordinate
+// one entry for core's y coordinate
+#define NUM_ENTRIES_PER_SHARD 3
+
+
+#define COMMAND_PTR_SHARD_IDX_OFFSET 6
```

Comment:
I moved `NUM_ENTRIES_PER_SHARD` into tt_metal/hostdevcommon/common_values.hpp
and `COMMAND_PTR_SHARD_IDX` into `tt_metal/hostdevcommon/common_runtime_address_map.h`
since both of these values are used in both the host and device code.